analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -474,12 +474,12 @@ def test_tidyargs_saargs(self):
         """
         build_json = os.path.join(self.test_workspace, "build_extra_args.json")
         report_dir = os.path.join(self.test_workspace, "reports_extra_args")
-        source_file = os.path.join(self.test_dir, "extra_args.c")
+        source_file = os.path.join(self.test_dir, "extra_args.cpp")
         tidyargs_file = os.path.join(self.test_dir, "tidyargs")
         saargs_file = os.path.join(self.test_dir, "saargs")
 
         build_log = [{"directory": self.test_dir,
-                      "command": "cc -c " + source_file,
+                      "command": "g++ -c " + source_file,
```

Comment:
I don't see any test cases where you use the `--analyzer-config` and `--tidyargs` in the same command. So please create a new test case or extend an existing one with it.

Reply:
Okay, I understand it but I would like to see a test for example where we set the `clang-tidy:Checks` and a `clang-tidy:HeaderFilterRegex` config option in the `--analyzer-config` option of the command line and in the --tidy-args I for example the Checks.